modules/signatures/windows/public_service_abuse.py

```diff
@@ -0,0 +1,49 @@
+# Copyright (C) 2012 Claudio "nex" Guarnieri (@botherder)
+#
+# This program is free software: you can redistribute it and/or modify
+# it under the terms of the GNU General Public License as published by
+# the Free Software Foundation, either version 3 of the License, or
+# (at your option) any later version.
+#
+# This program is distributed in the hope that it will be useful,
+# but WITHOUT ANY WARRANTY; without even the implied warranty of
+# MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+# GNU General Public License for more details.
+#
+# You should have received a copy of the GNU General Public License
+# along with this program.  If not, see <http://www.gnu.org/licenses/>.
```

Comment:
Please use the shorter copyright header from now on and also put in either `Copyright 2017 Cuckoo Foundation` (as found in the shorter copyright header) or something that attributes you :-)